src/accordion/accordion.ts

```diff
@@ -175,6 +175,44 @@ export class NgbAccordion implements AfterContentChecked {
     }
   }
 
+  /**
+   * Programmatically expand a panel with a given id if it is closed.
+   */
+  expand(panelId: string) {
+    const panel = this.panels.find(p => p.id === panelId);
+
+    if (panel && !panel.disabled && !panel.isOpen) {
+      this.toggle(panelId);
```

Comment:
IMO we should it differently - the logic of expanding should be in this method and `toggle` should call it based on the state. In other words - the `toggle` method should make use of the new `expand` / `collapse ` and not the other way around.